server/vitals.js

```diff
@@ -34,36 +34,39 @@ async function sendSingleAlert(locationid, message, pgClient) {
   })
 }
 
-async function sendDisconnectionMessage(locationid, displayName, language) {
+async function sendDisconnectionMessage(locationid, displayName, language, cDisplayName) {
```

Comment:
We (and in JS in general) tend to use more verbose variable names to make sure that there isn't any confusion down the line. Could you please change `cDisplayName` to `clientDisplayName` and now that there are two display names, it would also be a good time to update `displayName` to something more descriptive as well.

Same thing for `sendDisconnectionReminder` and `sendReconnectionMessage` too.

Reply:
thanks for the feedback! sounds good, i think i got into the habit of writing things for myself (favouring shorter names) which is why i did it, but yes thank you for pointing that out!